src/themes/creator.js

```diff
@@ -573,9 +577,6 @@ export default function createTheme(
     zIndex: {
       modal: 2000,
     },
-    tooltip: {
-      backgroundColor: primitives.mono900,
```

Comment:
do you think we can keep this here? I know this is inconsistent with the rest of the color values, but this would be a breaking change - I'd recommend adding a note here to change this in the next major

Reply:
Should I add it to the `ThemeT` type then? Because right now this field is incompatible with the specified type?

Reply:
oh, that's a good catch - yes, we should add it to the ThemeT type. also, can you make sure to add a comment too, that this is something we are removing in the next major?

Reply:
Done